enhancements/network/user-defined-network-segmentation.md

```diff
@@ -521,13 +520,25 @@ type UserDefinedNetworkSpec struct {
     // Supported by Topology `Layer2` and `Localnet`.
     // +optional
     IPAMLifecycle NetworkIPAMLifecycle `json:"ipamLifecycle,omitempty"`
+    
+    // Control how much of the IP/MAC-address configuration will be managed by OVN.
+    // `Full` means OVN-Kubernetes will apply IP configuration to the SDN infrastructure and it will also assign IPs from the selected subnet to the
+    // individual pods.
+    // `MAC` means OVN-Kubernetes will only assign MAC addresses and provide layer 2 communication, letting users configure IP addresses for the pods. 
+    //`MAC` is only available for `Layer2` and `Localnet` topologies for Secondary networks. 
+    // By disabling IPAM, any Kubernetes features that rely on selecting pods by IP will no longer function 
+    // (such as network policy, services, etc). Additionally, IP port security will also be disabled for interfaces attached to this network.
+    // Defaults to `Full`.
+    // +optional
+    AddressManagement AddrManagement `json:"addrManagement,omitempty"`
```

Comment:
```suggestion
    AddressManagement AddrManagement `json:"addressManagement,omitempty"`
```

I think it looks prettier. We are using full names everywhere else (except for IPAM), keeping this short may cause more harm (confusion) than good.

Reply:
lol that is actually what I intended to write, thank for catching!